js/data.js

```diff
@@ -12,6 +12,11 @@ function artDetails(objectNum) {
   xhrTwo.addEventListener('load', function () {
     console.log(xhrTwo.status);
     console.log(xhrTwo.response);
+    var artDescription = xhrTwo.response.artObject.plaqueDescriptionEnglish;
+    var DescriptionNode = document.querySelector('#viewedArt > p.art-description');
+
+    DescriptionNode.innerHTML = '<b>Description:</b> ' + artDescription;
```

Comment:
Instead of doing this with `innerHTML`, you should use DOM creation as we did it in class